### ISSUE TITLE
Search filters persist if navigating off the search view

### ISSUE THREAD

**Issue by terotic** [City-of-Helsinki/kerrokantasi-ui#388]
Filters should reset when leaving/entering the search view or the applied filters should be visible on the search form when returning to search view.

**Comment by jonnetanninen** [comment]
Should have been fixed at this point

**Comment by Rikuoja** [comment]
Not exactly. Seems that returning to the view after going to another view, the filters disappear from the URL and the form, but the hearing list is not reloaded when entering the view again.

Therefore, the hearing list remains filtered until the user clicks "Search" again.

**Comment by jonnetanninen** [comment]
Can't reproduce this :( Could you provide more specific case, e.g. after navigating to what view and way of navigating back? That might help me to reproduce this

**Comment by Rikuoja** [comment]
Yep. Just 1) select a label filter on the search page, 2) click "Tietoa palvelusta" and 3) click on "Kuulemiset" again. The filter is cleared, but the list still contains the filtered contents.
